apps/namadillo/src/hooks/useSdk.tsx

```diff
@@ -2,24 +2,30 @@ import initSdk from "@heliax/namada-sdk/inline-init";
 import { Sdk, getSdk } from "@heliax/namada-sdk/web";
 import { createStore } from "jotai";
 import { createContext, useContext, useEffect, useState } from "react";
-import { nativeTokenAtom, rpcUrlAtom } from "slices/settings";
+import { nativeTokenAddressAtom } from "slices/chainParameters";
+import { rpcUrlAtom } from "slices/settings";
 
 export const SdkContext = createContext<Sdk | null>(null);
 
 const initializeSdk = async (): Promise<Sdk> => {
   const { cryptoMemory } = await initSdk();
   const store = createStore();
```

Comment:
Doesn't need to be changed in this PR and I don't know if it's actually causing any problems yet, but we should probably use a global instance of `store` with a provider in `index.tsx`, otherwise atoms will end up reading different values depending on where they're called.

Reply:
I've replaced `createStore` by `getDefaultStore` , because we're not setting any store in the Provider